### ISSUE TITLE
Test Zurb Foundation for Apps on Internet Explorer

### ISSUE THREAD

**Issue by drernie** [TheSwanFactory/self-service-kiosk#9]
Just open the client HTML files in whatever browsers you have installed, and check that the pages reflow nicely when you resize the window.  Thanks!

https://github.com/TheSwanFactory/self-service-kiosk/tree/master/client

**Comment by huffmande** [comment]
Do you mean test the 3 html files (home.html, index.html, and question.html) at this location?:
https://github.com/TheSwanFactory/self-service-kiosk/tree/master/src

The link that refers to "/master/client" is broken - doesn't exist.  There isn't a "client" folder/directory.

**Comment by danielma** [comment]
Yeah, sorry about that @huffmande . I changed the source location a few pull-requests ago

**Comment by huffmande** [comment]
@danielma 
Okay .. thx.  No problem ... I know that it's a product still very much in development so it's going to change; just wanted to make sure I was assuming the correct new location.  Thanks for getting back to me.
